tree.go

```diff
@@ -439,6 +439,16 @@ func (n *InternalNode) CreatePath(path []byte, stemInfo stemInfo, comms []*Point
 			n.children[path[0]] = Empty{}
 		case extStatusAbsentOther:
 			// insert poa stem
+			newchild := &LeafNode{
+				commitment: comms[0],
+				stem:       stemInfo.stem,
+				values:     nil,
+				depth:      n.depth + 1,
+			}
+			n.children[path[0]] = newchild
+			comms = comms[1:]
+			newchild.c1 = new(Point)
+			newchild.c2 = new(Point)
```

Comment:
This mostly means "empty value", using the same behavior of `extStatusPresent`.

I'm not yet entirely convinced this is ideal, but maybe it is sufficient. Having a `new(Point)` is setting a value that a future "reader" will use as correct but this technically means "not known". But we don't have a `PartialLeafNode` type.

When I say "sufficient" I refer for a stateless client. This client will only need this LeafNode as a proof that a LeafNode with a different steam is here, instead of "something else" that is looking for. So, it _shouldn't_ ever really **need** to read C1/C2, which "is fine".

It's all quite subtle. I'm not sure it's worth adding another LeafNode type for that.

Note that this comment is unrelated with the fix; since the same comment applies with existing code in `extStatusPresent` (i.e: L466, etc below).